### ISSUE TITLE
Proper way to use this repository

### ISSUE THREAD

**Issue by valankar** [a5chin/python-uv#34]
Hi, thanks for making this. I'm pretty new to dev containers, but as far as I understand there are dev container templates that can be used to create projects on. Can I use this repo as a template somehow?

What's the proper way to use this dev container setup for a project? I *think* it is `Dev containers: Clone Repository in Container Volume` and then point to this Github repo. Then I replace the files with my code. I notice in this case VScode calls the container 'python-uv' but I'd like to call it something related to my project. I'm not sure how to rename it or where even that string comes from (probably github repo name).

Or do I just copy in the `.devcontainer` directory into my project and build a container? Anyhow, maybe a few sentences in the README might help to get started.

**Comment by a5chin** [comment]
@valankar 
Thank you for your issue!

You can use this repo as a template in the way you pointed out.
- Dev containers: Clone (or **fork** and rename) Repository in Container Volume
- point to this Github repo
- replace the files with my code

**Comment by valankar** [comment]
Thanks, I notice if I do this and create multiple containers, it ends up looking like this:

![image](https://github.com/user-attachments/assets/1631875e-5cad-40f4-90ad-f8e007d81d40)

i.e it becomes difficult to know what source tree is in each container. Also you cannot tell in File -> Open Recent either.

**Comment by a5chin** [comment]
I’m not sure what actions were taken or what went wrong. Could you please explain what the ideal outcome was, what steps you took, and what the actual result was?

**Comment by valankar** [comment]
I've made a screen capture. I'm not actually sure how I ended up with 2 python-uv containers in my screenshot. But here is an example of now trying to create 2 different containers. Only the first one is created, and trying another clone just ends up re-opening the first one.

https://youtu.be/wigVzIdr48A?si=OUcrKsoA28Tf7BmN

So basically I'm trying to figure out how to use this for multiple projects/source trees.
